### PR TITLE
Fix bad user browsing ID handle

### PR DESCRIPTION
**Pull request by christophehenry**

### CODE REVIEW

File: socialhome/streams/app/components/AuthorBar.vue [hunk 1]
```diff
@@ -84,7 +84,7 @@ export default Vue.component("author-bar", {
             this.showProfileBox = !this.showProfileBox
         },
         unfollow() {
-            if (!this.currentBrowsingProfileId) {
+            if (!this.isUserAuthentificated) {
```
Comment (jaywink) [inline socialhome/streams/app/components/AuthorBar.vue]: Should be `isUserAuthenticated`.

File: socialhome/streams/app/components/AuthorBar.vue [hunk 2]
```diff
@@ -93,7 +93,7 @@ export default Vue.component("author-bar", {
                 .catch(err => console.error(err) /* TODO: Proper error handling */)
         },
         follow() {
-            if (!this.currentBrowsingProfileId) {
+            if (!this.isUserAuthentificated) {
```
Comment (jaywink) [inline socialhome/streams/app/components/AuthorBar.vue]: Should be `isUserAuthenticated`.

**Review by jaywink — CHANGES_REQUESTED** [review]
Once tests pass, please rewrite commit message so represents a professional commit showing the reader what the commit is about, thanks ;)

**Comment by christophehenry** [comment]
Done

**Review by jaywink — APPROVED** [review]

**Comment by jaywink** [comment]
Excellent

**Pull request merged**